lib/checks/aria/deprecatedrole-evaluate.js

```diff
@@ -0,0 +1,21 @@
+import standards from '../../standards';
+import { getRole } from '../../commons/aria';
+
+/**
+ * Check that an elements semantic role is deprecated.
+ *
+ * Deprecated roles are taken from the `ariaRoles` standards object from the roles `deprecated` property.
+ *
+ * @memberof checks
+ * @return {Boolean} True if the elements semantic role is deprecated. False otherwise.
+ */
+export default function deprecatedroleEvaluate(node, options, virtualNode) {
+  const role = getRole(virtualNode, { dpub: true });
```

Comment:
Should we look at at each role segment instead of the semantic role?

```js
const roles = tokenList(virtualNode.attr('role'));
roles.forEach(role => {
  const roleDefinition = standards.ariaRoles[role];
  // ... 
});
```